test/dotnet-project-convert.Tests/DotnetProjectConvertTests.cs

```diff
@@ -569,6 +570,137 @@ public void Directives_Whitespace_Invalid()
             expectedWildcardPattern: "Invalid property name at /app/Program.cs:1. The '\t' character, hexadecimal value 0x09, cannot be included in a name.");
     }
 
+    /// <summary>
+    /// <c>#:</c> directives after C# code are ignored.
+    /// </summary>
+    [Fact]
+    public void Directives_AfterToken()
+    {
+        VerifyConversion(
+            inputCSharp: """
+                #:property Prop 1
+                #define X
+                #:property Prop 2
+                Console.WriteLine();
+                #:property Prop 3
+                """,
+            expectedProject: $"""
+                <Project Sdk="Microsoft.NET.Sdk">
+
+                  <PropertyGroup>
+                    <OutputType>Exe</OutputType>
+                    <TargetFramework>{ToolsetInfo.CurrentTargetFramework}</TargetFramework>
+                    <ImplicitUsings>enable</ImplicitUsings>
+                    <Nullable>enable</Nullable>
+                  </PropertyGroup>
+
+                  <PropertyGroup>
+                    <Prop>1</Prop>
+                    <Prop>2</Prop>
+                  </PropertyGroup>
+
+                </Project>
+
+                """,
+            expectedCSharp: """
+                #define X
+                Console.WriteLine();
+                #:property Prop 3
+                """);
+    }
+
+    /// <summary>
+    /// <c>#:</c> directives after <c>#if</c> are ignored.
+    /// </summary>
+    [Fact]
+    public void Directives_AfterIf()
+    {
+        VerifyConversion(
+            inputCSharp: """
+                #:property Prop 1
```

Comment:
Do we have tests for when the property value has characters that could impact XML processing? For example:

```cs
#:property Evil </Property>
```

Reply:
Yes, `Directives_Escaping`